src/main/kotlin/data/persistence/AppDatabase.kt

```diff
@@ -0,0 +1,15 @@
+package data.persistence
+
+import data.dao.Dao
+import data.model.*
+
+interface AppDatabase {
+    fun getUserDao(): Dao<User>
+    fun getLanguageDao(): Dao<Language>
```

Comment:
you have a LanguageDao and a UserDao now; returning a Dao<T> for these will reduce exposed APIs

Reply:
Based on our discussion on daos on Friday, we are going to remove `UserDao` and stick with `Dao<User>`, but you are correct, we need to add `LanguageDao`